### PR TITLE
add onepage website

### PR DESCRIPTION
**Pull request by AbbyNyakara**
In this project, we created a one-page navigation website for the awesome books project with fixed a header and footer. All the functionalities of the project remain the same.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
-Awesome Books 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```
Comment (brytebee) [inline README.md]: - [x] Kindly remove the placeholder `Project Name` and replace it with your current project title. This makes your project look more professional.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @AbbyNyakara & @aster-alemu👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Contains all three content sections.✔️
- When any of the links are clicked, the main area changes to display.✔️
- Awesome design.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AbbyNyakara) [reply README.md]: @brytebee , we have made the changes

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi team 👋, 

Well done making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights ⭐ 
✔️ Descriptive PR
✔️ App works as expected
✔️ Good collaboration
✔️ Linter checks

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**